crates/swc_ecma_transforms_base/src/helpers/mod.rs

```diff
@@ -321,17 +323,7 @@ impl VisitMut for Marker {
         i.span = i.span.apply_mark(self.0);
     }
 
-    fn visit_mut_member_prop(&mut self, p: &mut MemberProp) {
```

Comment:
Previously we don't visit object here.

Reply:
By that logic we could also remove some `visit_mut_prop_name`

Reply:
Yes, I think you are right.

Reply:
@kwonoj How do you think about this?